### PR TITLE
completed prep-css-flexbox Jacob Yi

### PR DESCRIPTION
**Pull request by Yi-Jacob**
Screen Shot 2022-01-14 at 10 02 04 AM
Screen Shot 2022-01-14 at 10 02 35 AM
Screen Shot 2022-01-14 at 10 03 21 AM

### CODE REVIEW

File: prep-css-flexbox/flex-wrap.html [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Flexbox - flex-wrap</title>
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="starwars.css">
+  <style>
+    #card-deck {
+      display: flex;
+      flex-wrap: wrap;
+      flex-basis: calc(100% /6);
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html]: You'll want to make sure that `flex-basis` is used on your `.card-wrapper`. Some flexbox properties are used on the "flex container" (parent) and others are used on the "flex items" (children).

File: prep-css-flexbox/flex-wrap.html [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+    }
+    .card-deck {
+      padding: 3px;
+      flex-basis: calc(100% /6)
+    }
```
Comment (thebearingedge) [inline prep-css-flexbox/flex-wrap.html]: This rule set has a selector for all elements with the `class="card-deck"` attribute, but we're only putting `id="card-deck"` on the parent and `class="card-wrapper"` on the children, so you can remove this rule set.

**Review by thebearingedge — CHANGES_REQUESTED** [review]

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-14 at 1 07 19 PM

**Comment by Yi-Jacob** [comment]
Screen Shot 2022-01-14 at 1 07 19 PM

**Review by thebearingedge — APPROVED** [review]

**Pull request merged**